.github/workflows/pytest-rts.yml

```diff
@@ -30,3 +30,46 @@ jobs:
         run: |
           pip install tox
           tox -e py -- --exitfirst
+
+  evaluation:
+    needs: test
+    name: Run evaluation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone target repo
+        run: |
+          cd ../../.
+          git clone https://github.com/willmcgugan/rich.git
```

Comment:
Shall we use `matrix` of selected projects here? Same way we use `matrix` of python versions above.